web/client/src/components/form/TokenCredentials.jsx

```diff
@@ -100,7 +100,7 @@ export default function TokenCredentials(props) {
               accept=".json"
               onChange={handleFile}
             />
-            <Button color="primary" variant="contained" component="span">
+            <Button variant="contained" component="span">
```

Comment:
Not sure if we should be removing the color from the button entirely, but we can add a different color later I guess.